crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -27,6 +27,7 @@ class ApplicationController < ActionController::Base
 
   before_action do |c|
     Crowbar::Sanity.cache! unless Rails.cache.exist?(:sanity_check_errors)
+    check_eol unless Rails.cache.fetch(:eol_checked).blank?
```

Comment:
Doesn't the `unless ... blank?` make sure that the `check_eol` is actually never called?